templates/screen-capture.php

```diff
@@ -14,6 +14,26 @@
 page.settings.resourceTimeout = <?php echo $timeout ?>;
 <?php endif ?>
 
+<?php if (isset($blockAnalytics) && $blockAnalytics == true) : ?>
+page.onResourceRequested = function(requestData, request) {
+	if ((/google-analytics\.com/gi).test(requestData['url'])){
+		request.abort();
+	}
+	//
+	if ((/track.hubspot.com/gi).test(requestData['url'])){
+		request.abort();
+	}
+	// Lucky Orange Tracking
+	if ((/luckyorange\.com/gi).test(requestData['url'])){
+		request.abort();
+	}
+	// Lucky Orange Tracking
```

Comment:
Thank you for your submission @Failcookie 

I noticed that the comments indicating the tracking system that will be blocked are not fully correct, can you fix them please?